.circleci/config.yml

```diff
@@ -24,10 +24,39 @@ jobs:
           path: /home/circleci/project/tests/test-reports
       - store_artifacts:
           path: /home/circleci/project/htmlcov
+  build_docker:
+    environment:
+      IMAGE_ACCOUNT: rfaircloth
```

Comment:
I do not think it would be suitable to add your Docker build process in the EventgenCI. I would recommend creating a separate CI pipeline that pulls from this repo for your personal use case.